src/redirectPage.tsx

```diff
@@ -84,7 +84,7 @@ export default function RedirectPage (): JSX.Element {
         <h3 className="">{displayString}</h3>
         <ServiceWorkerReadyButton id="load-content" label='Load content' waitingLabel='Waiting for service worker registration...' onClick={() => { window.location.href = reloadUrl }} />
       </div>
-      <ConfigIframe />
+      {showConfigIframe && <ConfigIframe />}
```

Comment:
for path based requests in `first-hit` scenario, we don't want to show the iframe (because the request is on the rootDomain), but we do want to show the "waiting for service worker.." etc code.

ideally, we should show the config page *outside* of an iframe, but I'm open to ideas @lidel @2color